BAZProject/MainMovie/MainMovieProtocols.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  MainMovieProtocols.swift
+//  BAZProject
+//
+//  Created 1030364 on 18/10/22.
+//  Copyright © 2022 ___ORGANIZATIONNAME___. All rights reserved.
+//
+//  Template generated by Juanpe Catalán @JuanpeCMiOS
+//
+
+import Foundation
+
+// MARK: Wireframe -
+protocol MainMovieWireframeProtocol: AnyObject {}
+
+// MARK: Presenter -
+protocol MainMoviePresenterProtocol: AnyObject {
```

Comment:
Siguiendo los principios de Clean Code, es mejor usar el sufijo Imp para nombrar las implementaciones, que el sufijo Protocol para nombrar los protocolos.

Por lo tanto te recomendaría renombrar este protocolo así:
```suggestion
protocol MainMoviePresenter: AnyObject {
```

Y la implementación debería ser algo así:
`final class MainMoviePresenterImp: MainMoviePresenter`